pkg/descheduler/strategies/pod_lifetime.go

```diff
@@ -49,6 +49,13 @@ func PodLifeTime(ctx context.Context, client clientset.Interface, strategy api.D
 		klog.V(1).Info(err)
 		return
 	}
+	priorityClass := strategy.Params.ThresholdPriorityClassName
+	priority, err := podEvictor.SetPriorityThresholdFromPriorityClass(ctx, client, priorityClass)
```

Comment:
any reason to not pass `strategy.Params.ThresholdPriorityClassName` directly to the function?